.github/workflows/rust.yml

```diff
@@ -0,0 +1,26 @@
+name: Rust
+
+on:
+  push:
+    branches: [ iroha2-dev ]
+  pull_request:
+    branches: [ iroha2-dev ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Format
+      run: cargo fmt -- --check
+    - name: Build
+      run: cargo build --verbose
+    - name: Run tests
+      run: cargo test --verbose
+    - name: rust-clippy-check
```

Comment:
I think clippy check will be faster then build and test, so maybe move it after the format and before build step.